### PR TITLE
Hashtag lower

### PR DESCRIPTION
**Pull request by mikeyrayvon**
make hashtags lowercase

### CODE REVIEW

**Comment by mixcas** [comment]
Im passing this to you, @varffs

File: app/imports/api/photosMethods.js [hunk 1]
```diff
@@ -21,31 +21,36 @@ export const addPost = new ValidatedMethod({
...
+      let sanitizedCaption = sanitizeHtml(caption);
+
+      // Add to `data`
+      data.caption = sanitizedCaption;
+
+      // Parse hastags
+      const hashtagsArray = sanitizedCaption.match(/#\S+/g);
+
+      // lowercase the hastags
+      hashtagsArrayLower = _.map(hashtagsArray, hashtag => hashtag.toLowerCase());
```
Comment (varffs) [inline app/imports/api/photosMethods.js]: this is missing var declaration. `hashtagsArray` could become a `let` and then be overwritte

File: app/imports/api/photosMethods.js [hunk 2]
```diff
@@ -21,31 +21,36 @@ export const addPost = new ValidatedMethod({
...
+      let sanitizedCaption = sanitizeHtml(caption);
+
+      // Add to `data`
+      data.caption = sanitizedCaption;
+
+      // Parse hastags
+      const hashtagsArray = _.compact(sanitizedCaption.match(/#\S+/g));
+
+      // lowercase the hastags
+      hashtagsArrayLower = _.map(hashtagsArray, hashtag => hashtag.toLowerCase());
```
Comment (varffs) [inline app/imports/api/photosMethods.js]: missing `var` declaration. `hashtagsArray` could be a `let` and have its value overwritten here. Same for above in comments method

**Review by varffs — CHANGES_REQUESTED** [review]

**Comment by varffs** [comment]
Want me to do the edits?

**Review by varffs — APPROVED** [review]

**Pull request merged**